index.js

```diff
@@ -70,14 +70,19 @@ function parseDependencies (source, root) {
  * output transformed source.
  */
 function transformSource (runner, config, source, map, callback) {
+  var subprocessEnv = {
+    stdio: ['pipe', 'pipe', process.stderr],
+    env: defaults({}, config.env || {}, process.env)
```

Comment:
So just `env: config.env || process.env` here.

Reply:
Oops, not that... just `config.env` here, and instead specify `process.env` in [here](https://github.com/usabilityhub/rails-erb-loader/blob/c2d980199eba59ceb7d8401194c4992329ec40a2/index.js#L191-L195) where default options are specified.